app/src/main/java/com/github/epfl/meili/map/MapActivity.kt

```diff
@@ -24,9 +24,9 @@ import com.google.android.libraries.places.api.net.PlacesClient
 
 class MapActivity : AppCompatActivity(), OnMapReadyCallback {
     companion object {
-        private val TAG = MapActivity::class.java.name
         private const val DEFAULT_ZOOM = 15
         private const val REQUEST_CODE: Int = 1
+        const val POI_KEY = "POI_KEY"
```

Comment:
Could you please explain what the POI_KEY is supposed to represent?

Reply:
It's used as a key when passing the Intent (which contains the poi as a value) to the PoiActivity which is launched when a poi on the map is clicked.

Reply:
One question, in the ChatGroup section we expect to get a uniquely identifier for the POI clicked. Do you think this could the best place to give this uid in the intent? If yes, are you planning to do it as a follow up? If no, where do you think it would be better to do this?

Reply:
I think it's ok to send the entire POI for now since I also use the POI name to show at the top of the chat log activity. Also, we might want to be able to show other features of the POI on the chat interface later.

Reply:
@MarcelTorne Yes I think this is the right place, and actually Google Map's PointOfInterest class already contains such a unique id as a field. Ad @ahmedc3ca says, I am passing the entire POI here so that we can also access its name, etc. as needed.

Reply:
That sounds really good, thank you!